revy.yml

```
@@ -0,0 +1,73 @@
+apiVersion: reviewpad.com/v0.0.2
+
+mode: silent
+
+labels:
+  small:
+    description: small changes
+    # color is the hexadecimal color code for the label, without the leading #.
+    color: "294b69"
+  medium:
+    description: small changes
+    color: "a8c3f7"
+  large:
+    description: large changes
+    color: "8a2138"
+
+rules:
+  isSmall:
+    kind: patch
+    description: small pull request
+    spec: $size() <= 90
+
+  isMedium:
+    kind: patch
+    description: medium-sized pull request
+    spec: $size() > 90 && $size() <= 300
+
+  isLarge:
+    kind: patch
+    description: large-sized pull request
+    spec: $size() > 300
+  
+  tautology:
+    kind: patch
+    description: true by default
+    spec: "true"
+
+  touchesContracts:
+    kind: patch
+    description: touches package contracts
+    spec: $hasFilePattern("packages/contracts/**")
+
+protectionGates:
+  - name: assign-naps-for-everything
+    description: By default, assign naps for every pull request
+    patchRules:
+      - rule: tautology
```

```suggestion
      - rule: touchesContracts
```

I think this is actually what we want?
I don't want to be assigned reviews to `packages/web` :smile:

---

Oh, I actually forgot it was setup like that (it was made before the frontend team joined)

but yes, I'd rather keep only packages/contracts for me, at least for now